src/main/java/com/jfrog/ide/idea/scan/ScanManagersFactory.java

```diff
@@ -113,6 +107,21 @@ public void startScan(boolean quickScan) {
         }
     }
 
+    private boolean refreshConnection() {
```

Comment:
Let's add a description for method. Specifically explain the boolean returned value.